docs/sources/alerting/configure-notifications/mute-timings.md

```diff
@@ -37,51 +38,55 @@ refs:
+## Mute and active timings vs silences
 
-[Mute timings](ref:shared-mute-timings) and [silences](ref:shared-silences) are distinct methods to suppress notifications. They do not prevent alert rules from being evaluated or stop alert instances from appearing in the user interface; they only prevent notifications from being created.
+[Mute and active timings](ref:shared-mute-timings) and [silences](ref:shared-silences) are distinct methods to suppress notifications. They do not prevent alert rules from being evaluated or stop alert instances from appearing in the user interface; they only prevent notifications from being created.
 
-The following table highlights the key differences between mute timings and silences.
+The following table highlights the key differences between mute and active timings, and silences.
 
-|            | Mute timing                                                 | Silence                                                          |
+|            | Mute and active timing                                      | Silence                                                          |
 | ---------- | ----------------------------------------------------------- | ---------------------------------------------------------------- |
 | **Setup**  | Created and then added to notification policies             | Matches alerts using labels to determine whether to silence them |
 | **Period** | Uses time interval definitions that can repeat periodically | Has a fixed start and end time                                   |
 
 [//]: <> ({{< docs/shared lookup="alerts/mute-timings-vs-silences.md" source="grafana" version="<GRAFANA_VERSION>" >}})
 
-## Add mute timings
+## Add time intervals
 
 1. In the left-side menu, click **Alerts & IRM**, and then **Alerting**.
-1. Click **Notification policies** and then the **Mute Timings** tab.
+1. Click **Notification policies** and then the **Time intervals** tab.
 1. From the **Alertmanager** dropdown, select an external Alertmanager. By default, the **Grafana Alertmanager** is selected.
-1. Click **+ Add mute timing**.
-1. Fill out the form to create a [time interval](#time-intervals) to match against for your mute timing.
-1. Save your mute timing.
+1. Click **+ Add time interval**.
+1. Fill out the form to create a [time interval](#time-intervals) to match against for your mute or active timing.
+1. Save your changes.
 
-## Add mute timing to a notification policy
+## Add notification timing to a notification policy
 
 1. In the left-side menu, click **Alerts & IRM**, and then **Alerting**.
 1. Click **Notification policies** and make sure you are on the **Notification Policies** tab.
-1. Find the notification policy you would like to add the mute timing to and click **...** -> **Edit**.
-1. From the **Mute timings** dropdown, choose the mute timings you would like to add to the policy.
+1. Find the notification policy you would like to add the notification timing to and click **...** -> **Edit**.
```

Comment:
term "notification timing" seems to be new here. Let's stick with "time intervals"?